src/snowflake/snowpark/functions.py

```diff
@@ -4096,8 +4532,24 @@ def ntile(e: Union[int, ColumnOrName]) -> Column:
     Divides an ordered data set equally into the number of buckets specified by n.
     Buckets are sequentially numbered 1 through n.
 
-    Args:
-        e: The desired number of buckets; must be a positive integer value.
```

Comment:
why are we removing arg definition?

Reply:
it must be removed by mistake, let me add it back